### PR TITLE
Renders html for in product help xp

### PR DESCRIPTION
**Pull request by caylahamann**
Provides transformations to the html to be able to be used by the in product help experience. Adopts some of the old class names from the legacy docs site, as well as some new class names to allow for better handling on the in product help experience side. 

Next steps: 
* [ ] ensure this format renders correctly for the in product help xp 
* [ ] make a PR for the docs library service to consume this json end-point

partially addresses #444

### CODE REVIEW

**Comment by jerelmiller** [comment]
qq: Are we using feature flags to handle the switchover? We should be able to render from either site on the in product help so that we aren't hindered on what we can output from here 🙂

**Comment by jerelmiller** [comment]
One more qq: What do the image URLs end up being? Since we are compiling to html on our own, I'd be curious if the images are outputting with the compiled image paths or the relative image from the raw markdown.

**Comment by caylahamann** [comment]
> qq: Are we using feature flags to handle the switchover? We should be able to render from either site on the in product help so that we aren't hindered on what we can output from here 🙂

Goodpoint!! I will figure out how to use feature flags and will implement that into my work :)

**Comment by caylahamann** [comment]
> One more qq: What do the image URLs end up being? Since we are compiling to html on our own, I'd be curious if the images are outputting with the compiled image paths or the relative image from the raw markdown.

Also a great point. They end up being relative to the markdown, so I will need to figure out a way to get the relative image path for them. Do you have any ideas on how to do this?

**Comment by caylahamann** [comment]
This now gets the static image location :) 
example: 
```
{"body":"<p><strong>Account Owner or Administrators:</strong> Before adding <a href=\"http://campfirenow.com/\" title=\"Link opens in a new window\">Campfire</a> notifications to New Relic, you must have an existing Campfire account with the following information:</p>\n<ul>\n<li>Subdomain</li>\n<li>Email</li>\n<li>Authentication token</li>\n<li>Campfire room names as applicable</li>\n</ul>\n<h2 id=\"qiklinks\">Contents </h2>\n<h2 id=\"channel\">Creating a Campfire channel </h2>\n<p><img src=\"/static/ae9be9bbaab5abb51439e66bebcd416a/73c85/screen-alert-channel-campfire_0.png\" alt=\"screen alert channel campfire.png\"></p>\n<p><strong>Alerts > Alert policies > Channels and groups:</strong> Your New Relic account's Owner or Admins can integrate your Campfire account as an alert notification channel.</p>\n<p>To integrate your Campfire account with New Relic:</p>\n<ol>\n<li>From the New Relic menu bar, select <strong>Alerts > Alert policies > Channels and groups</strong>.</li>\n<li>From the Channels section, select <strong>Create channel > Campfire</strong>.</li>\n<li>Provide the information for your Campfire account.</li>\n<li>Select your <a href=\"/docs/alerts/alert-policies/configuring-alerts/alert-channels-maintaining-notification-lists#levels\">notification level</a>.</li>\n<li>Select <strong>Integrate with Campfire</strong>.</li>\n</ol>\n<p><strong>Tip:</strong> Fields have tooltip links to help you retrieve information from Campfire.</p>\n<h2 id=\"updates\">Updating Campfire channels </h2>\n<p>Follow standard procedures to <a href=\"/docs/alerts/alert-policies/configuring-alerts/alert-channels-maintaining-notification-lists#edit\">edit</a> or <a href=\"/docs/alerts/alert-policies/configuring-alerts/alert-channels-maintaining-notification-lists#delete\">delete</a> alert channels.</p>"}{"body":"<p><strong>Account Owner or Administrators:</strong> Before adding <a href=\"http://campfirenow.com/\" title=\"Link opens in a new window\">Campfire</a> notifications to New Relic, you must have an existing Campfire account with the following information:</p>\n<ul>\n<li>Subdomain</li>\n<li>Email</li>\n<li>Authentication token</li>\n<li>Campfire room names as applicable</li>\n</ul>\n<h2 id=\"qiklinks\">Contents </h2>\n<h2 id=\"channel\">Creating a Campfire channel </h2>\n<p><img src=\"/static/ae9be9bbaab5abb51439e66bebcd416a/73c85/screen-alert-channel-campfire_0.png\" alt=\"screen alert channel campfire.png\"></p>\n<p><strong>Alerts > Alert policies > Channels and groups:</strong> Your New Relic account's Owner or Admins can integrate your Campfire account as an alert notification channel.</p>\n<p>To integrate your Campfire account with New Relic:</p>\n<ol>\n<li>From the New Relic menu bar, select <strong>Alerts > Alert policies > Channels and groups</strong>.</li>\n<li>From the Channels section, select <strong>Create channel > Campfire</strong>.</li>\n<li>Provide the information for your Campfire account.</li>\n<li>Select your <a href=\"/docs/alerts/alert-policies/configuring-alerts/alert-channels-maintaining-notification-lists#levels\">notification level</a>.</li>\n<li>Select <strong>Integrate with Campfire</strong>.</li>\n</ol>\n<p><strong>Tip:</strong> Fields have tooltip links to help you retrieve information from Campfire.</p>\n<h2 id=\"updates\">Updating Campfire channels </h2>\n<p>Follow standard procedures to <a href=\"/docs/alerts/alert-policies/configuring-alerts/alert-channels-maintaining-notification-lists#edit\">edit</a> or <a href=\"/docs/alerts/alert-policies/configuring-alerts/alert-channels-maintaining-notification-lists#delete\">delete</a> alert channels.</p>"}
```

File: plugins/gatsby-plugin-generate-doc-json/utils/handlers.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  node.attributes
+    ? node.attributes.reduce((obj, { name: propName, value }) => {
+        return { ...obj, [propName]: value };
+      }, {})
+    : {};
+
+module.exports = {
+  image: (h, node, imageHashMap, fileRelativePath) => {
+    const srcUrl = fileRelativePath
+      .replace('.mdx', '')
```
Comment (caylahamann) [inline plugins/gatsby-plugin-generate-doc-json/utils/handlers.js:19]: this assumes that the relativePath will always be under `./images/` for the file image

**Review by zstix — APPROVED** [review]
I don't have a _ton_ of context on this stuff, but from what I can see this looks good!

**Pull request merged**